CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
+## [Unreleased]
+
+### Fixed
+* Fixed colliders on diagonal walls to not have negative scale.
```

Comment:
Just nitpicking to add the commit # for hyperlinking.

Reply:
I knew I was forgetting something! Will do :)